moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -23,6 +23,16 @@ servo:
     }
   }
 
+  latency: {
```

Comment:
I'd give this a more specific name than just "latency".

Reply:
Okay, I changed it to `max_expected_latency`